lms/djangoapps/static_template_view/urls.py

```diff
@@ -0,0 +1,57 @@
+"""
+URLs for static_template_view app
+"""
+
+from django.conf import settings
+from django.conf.urls import patterns, url
+
+urlpatterns = (
+    'static_template_view.views',
+
+    # TODO: Is this used anymore? What is STATIC_GRAB?
+    url(r'^t/(?P<template>[^/]*)$', 'index'),
+
+    # Semi-static views (these need to be rendered and have the login bar, but don't change)
+    url(r'^404$', 'render', {'template': '404.html'}, name="404"),
+    # display error page templates, for testing purposes
+    url(r'^404$', 'render_404'),  # Can this be deleted? Test test_404_microsites fails with this.
```

Comment:
This is probably needed so that microsites can redirect to a different 404 page.